pootle/static/js/shared/utils/relativeTime.js

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright (C) Pootle contributors.
+ *
+ * This file is a part of the Pootle project. It is distributed under the GPL3
+ * or later license. See the LICENSE file for a copy of the license and the
+ * AUTHORS file for copyright and authorship information.
+ */
+
+import { nt, t } from './i18n';
+
+
+/**
+ * A version of `Math.trunc` which returns absolute values.
+ *
+ * @param {Number} number - the number to truncate.
+ * @return {Number} - positive integer
+ */
+function absTrunc(number) {
```

Comment:
Oh, good catch, this is a copying & pasting remnant. I removed it, and also adjusted the relevant import statements.